### ISSUE TITLE
vmd with FastPBC 

### ISSUE THREAD

**Issue by groponp** [jvermaas/vmd-packaging-instructions#4]
Dear prof Vermaas,  I hope that you are fine. 

I want ask you, How do I install vmd with FastPBC? . I have review your README here, but it mention any lines from code that I am not sure if are the corrects. 

tcl_commands.C  I found  the following section containing fastpbc name, Do I change from 0 to 1? 
```C++
#if 0
  Tcl_CreateObjCommand(interp, "fastpbc", obj_fastpbc,
                       (ClientData) app, (Tcl_CmdDeleteProc *) NULL);
#endif
``` 

now in same file in line 283 I found , 
```C++
282 Tcl_CreateCommand(interp, "vmd_menu", text_cmd_menu,
 283       (ClientData) app, (Tcl_CmdDeleteProc *) NULL);
```

In this file (tcl_commands.C) I don't found more code that refer to FastPBC 

For other hand, I want to ask you, Is this fastPBC suitable to box with cubic and rectangular geometries? 
I need any very fast tool that PBCTools (the tcl program in VMD), because I have a 25K frames with 600K atoms, and a lot of replicas (>18) , and I only have workstation to perform this calculations.  I have try using qwrap, too, but this less flexible. 

Thanks so much in advanced for your help.

**Comment by jvermaas** [comment]
Yup, you flip the 0 to a 1 and recompile, and then the C preprocessor won't yank out that code. Fastpbc is similar to qwrap https://github.com/jhenin/qwrap, but can use GPUs.

**Comment by groponp** [comment]
@jvermaas  thanks so much, I would can try it.
